modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ViewBuilderScreen.scss

```
@@ -4,3 +4,7 @@
 	justify-content: center;
 	text-align: center;
 }
+
+.object-web-builder-screen-search-icon {
```

Generic class naming can cause name collision with other CSS classes

Adopt the [Block Element Modifier - BEM methodology](http://getbem.com/introduction/) to try avoid that.

```suggestion
.lfr-object__object-view-builder-search-icon {
```